packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.spec.ts

```diff
@@ -850,7 +850,7 @@ describe("DefaultVisualizers unit tests", () => {
 
 		const expected = {
 			fluidObjectId: "test",
-			nodeKind: "FluidTreeNode",
+			nodeKind: "FluidUnknownObjectNode",
```

Comment:
My guess is that the schema changed because we're now encoding when we take the snapshot, but I'm not sure if this is the correct fix to make.

Reply:
@jikim-msft since you worked a lot on the visualizer recently, any thoughts?

Reply:
this looks like a bug in the dev tools. they are not handling handles in their serialize form, but the code operates on the serialized content. @jzaffiro  i'd file a bug against dev tools for this.

Reply:
AB#7840 to track this issue

Reply:
As @anthony-murphy, this is happening due to the change in the `contentSnapshot()` now returning encoded format for handle node. 

Visualization logic in `core` (and possibly `view` package) side should change in order to understand the encoded value as a handle node type.